exchanges/auth/src/authExchange.test.ts

```diff
@@ -40,7 +43,10 @@ const makeExchangeArgs = () => {
           tap(op => operations.push(op)),
           map(result)
         ),
-      client: new Client({ url: '/api' }),
+      client: new Client({
+        url: '/api',
+        exchanges: [dedupExchange, cacheExchange, fetchExchange],
```

Comment:
TODO: remove exchanges, as we noticed in #3043 we are dispatching requests that error